src/utils/helpers.js

```diff
@@ -0,0 +1,17 @@
+export function setButtonText(
+  button,
+  isLoading,
+  defaultText = "Save",
+  loadingText = "Saving..."
+) {
+  if (isLoading) {
+    button.textContent = loadingText;
+  } else {
+    button.textContent = defaultText;
+  }
+}
+
+export function disableButton(button, settings) {
+  button.disabled = true;
+  button.classList.add(settings.inactiveButtonClass);
```

Comment:
Please, delete `disableButton` from `helpers.js` because you have it in `validation.js`. Please, export it from there

Reply:
* My remark hasn't been corrected https://github.com/ChrisLaFumee/se_project_spots/pull/1#discussion_r1985717694  Please, delete `disableButton` from `helpers.js` because you have it in `validation.js`. Please, export it from there